### ISSUE TITLE
MSW plugin: ERR_INVALID_ARG_TYPE

### ISSUE THREAD

**Issue by federico-ntr** [webpro/knip#562]
Whenever I try to run knip I get a TypeError: `TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Array`

<details>
  <summary>Complete stack trace</summary>

```bash
node:internal/validators:162
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
          ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Array
    at validateString (node:internal/validators:162:11)
    at join (node:path:1175:7)
    at file:///Users/project/node_modules/knip/dist/plugins/msw/index.js:11:48
    at Array.map (<anonymous>)
    at Object.resolveEntryPaths (file:///Users/project/node_modules/knip/dist/plugins/msw/index.js:11:18)
    at WorkspaceWorker.findDependenciesByPlugins (file:///Users/project/node_modules/knip/dist/WorkspaceWorker.js:198:67)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async main (file:///Users/project/node_modules/knip/dist/index.js:146:108)
    at async run (file:///Users/project/node_modules/knip/dist/cli.js:25:73)
    at async file:///Users/project/node_modules/knip/dist/cli.js:83:1 {
  code: 'ERR_INVALID_ARG_TYPE'
}
```
</details>

**Comment by webpro** [comment]
Can you please create a reproduction so I can see/debug it?

**Comment by webpro** [comment]
What's the value of `msw.workerDirectory` in `package.json`? Is it valid to be an array?

**Comment by federico-ntr** [comment]
The problem is indeed in `msw.workerDirectory` being an array. Looking at [this `msw` example repo](https://github.com/mswjs/examples/blob/c52e330bc60694c9fca9e334b7ada778f6650fe9/examples/with-vue/package.json#L36) it is a string, but `npx msw init public/` (which they suggest to setup msw) results in:

```json
  "msw": {
    "workerDirectory": [
      "public"
    ]
  }
```

So I guess it's a supported use case(?)

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.2.1. See [Release 5.2.1](https://github.com/webpro/knip/releases/tag/5.2.1) for release notes._

**Comment by webpro** [comment]
Alright then!
